src/main/java/ogc/rs/apiserver/router/routerbuilders/StacRouterBuilder.java

```diff
@@ -90,6 +90,24 @@ void addImplSpecificRoutes() {
         .handler(apiServerVerticle::buildResponse)
         .failureHandler(failureHandler);
 
+    routerBuilder
+        .operation(STAC_ITEMS_API)
+        .handler(stacAssetsAuthZHandler)
```

Comment:
Does not need the authZ handler

Reply:
The Items API should be sitting behind the AuthZ, no?

Reply:
@mahimatics should STAC `/items` and `/items/<itemId>` be under authZ? In my opinion, they shouldn't. I think that STAC Items are not same as OGC Feature items in this respect, because OGC feature items are data, whereas  STAC Items are a reference to the actual data which are the item assets.

Reply:
@ThorodanBrom  That is true, but the Items have some critical information in their properties as well as information such as file size of an asset and such. Moreover, the Items API is an extended path from `collection`  which in our DX environment is a resource, where the AuthZ is applicable, so if a collection is a `secure` one then the Items API should also be under AuthZ. This is my understanding.

Reply:
Remove authZ and audit